pkg/agent/route/route_linux.go

```diff
@@ -294,6 +294,18 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet string) *bytes
 			}...)
 		}
 	}
+
+	// To make liveness/readiness probe traffic bypass ingress rules of Network Policies, mark locally generated packets
+	// that will be sent to OVS so we can identify them later in the OVS pipeline.
+	// It must match source address because kube-proxy ipvs mode will redirect ingress packets to output chain, and they
+	// will have non local source addresses.
+	writeLine(iptablesData, []string{
```

Comment:
that will be before SNAT right?

I noticed that you are adding this to the raw table, and not in the mangle table, as you described in the issue (https://github.com/vmware-tanzu/antrea/issues/280#issuecomment-767008601). Intentional?

Reply:
Yes, SNAT is performend in POSTROUTING, we mark it in OUTPUT so it's before it.
Both raw and mangle can work, moved it to mangle in latest patch as raw is used mainly for configuring exemptions from connection tracking in combination with the NOTRACK target, while mangle is used for specialized packet alteration and we already have a rule in mangle to mark packets.